applications/dcc_decoder/README.md

```diff
@@ -0,0 +1,57 @@
+# DCC decoder application
```

Comment:
I would prefer this to be in Doxygen so that it can hyperlink to the relevant code as needed.  Maybe it is possible for this to remain in mark down and for Doxygen to pick it up.  https://www.stack.nl/~dimitri/doxygen/manual/markdown.html

Reply:
I added the applications directory but excluded all c and cxx sources from there to avoid the duplicate documentations/definitions.
I also figured out how to link this .md file into a meaningful place in our generated documentations.